.github/workflows/tests-php-with-woo-latest.yml

```diff
@@ -156,31 +156,44 @@ jobs:
-
-      # ------------------------------------------------------------------------------
-      # Verify artifacts exist
-      # ------------------------------------------------------------------------------
-      - name: Verify artifacts exist
+          cd ${GITHUB_WORKSPACE}/../the-events-calendar/common
+          HEAD_BRANCH="task/update-as-for/${{ github.head_ref }}"
+          git config --global user.email "actions@github.com"
+          git config --global user.name "github-actions"
+          git checkout -b "$HEAD_BRANCH"
+          echo "Created branch $HEAD_BRANCH"
+          echo "<?php do_action( 'plugins_loaded' ); echo ActionScheduler_Versions::instance()->latest_version();" > as_version.php
+          ${SLIC_BIN} site-cli plugin activate woocommerce
+          ${SLIC_BIN} site-cli eval-file as_version.php > as_version.txt
+          WC_AS_VERSION=$(grep -E '^[0-9]+\.[0-9]+\.[0-9]+' as_version.txt)
+          rm as_version.php as_version.txt
+          echo "Updating Action Scheduler to $WC_AS_VERSION"
+          ${SLIC_BIN} composer require woocommerce/action-scheduler:$WC_AS_VERSION
+          sed -i -E "s/[0-9]+\.[0-9]+\.[0-9]+/${WC_AS_VERSION}/g" tests/as_integration/Ensure_Latest_AS_Test.php
+          git add composer.json composer.lock tests/as_integration/Ensure_Latest_AS_Test.php
+          git commit -m "Update Action Scheduler to $WC_AS_VERSION"
+          git push origin "$HEAD_BRANCH"
+      - name: Create Pull Request using GitHub CLI
         if: ${{ failure() }}
+        env:
+          # Use GITHUB_TOKEN so that gh automatically authenticates.
+          GITHUB_TOKEN: ${{ secrets.GHA_BOT_TOKEN_MANAGER }}
         run: |
-          echo "Listing contents of '${GITHUB_WORKSPACE}/tests/_output/':"
-          ls -la "${GITHUB_WORKSPACE}/tests/_output/" || echo "No artifacts found."
+          echo "Creating PR using gh CLI..."
+          # Define the head branch name for the PR.
+          HEAD_BRANCH="task/update-as-for/${{ github.head_ref }}"
 
-      # ------------------------------------------------------------------------------
-      # Upload artifacts (On failure)
-      # ------------------------------------------------------------------------------
-      - name: Upload artifacts
-        uses: actions/upload-artifact@v4
-        if: ${{ failure() }}
-        with:
-          name: output ${{ matrix.suite }}
-          path: tests/_output/
-          retention-days: 7
+          # Create the PR and capture JSON output.
+          PR_URL=$(gh pr create \
+            --base "${{ github.head_ref }}" \
+            --head "$HEAD_BRANCH" \
+            --title "[BOT] Update AS Version to match WC" \
+            --body "This is an automated PR created by ${{ github.actor }} for [PR](${{ github.event.pull_request.url }}). It was generated by [this GitHub Action](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}) \\n [skip-changelog] [skip-lint] [skip-phpcs]" \
```

Comment:
This is not formatting correctly, see https://github.com/the-events-calendar/tribe-common/pull/2403